addons/account/models/account_move.py

```diff
@@ -423,7 +430,7 @@ def _reverse_move(self, date=None, journal_id=None, auto=False):
                 'date': date,
                 'journal_id': journal_id.id if journal_id else self.journal_id.id,
                 'ref': (_('Automatic reversal of: %s') if auto else _('Reversal of: %s')) % (self.name),
-                'auto_reverse': False})
+            })
```

Comment:
`auto` parameter should be removed and some of the above code as well, as it is now useless